community/concepts/grain.md

```diff
@@ -0,0 +1,73 @@
+### Grain
```

Comment:
This should be h1 to match others.

Reply:
Done. Changed the sub-heading to h2 too.